src/PowerShell/Help/Microsoft.WinGet.Client/Reset-WinGetSource.md

```diff
@@ -15,17 +15,25 @@ Resets default WinGet sources.
 ## SYNTAX
 
 ```
-Reset-WinGetSource -Name <String> [<CommonParameters>]
+Reset-WinGetSource [-Name <String>] [<CommonParameters>]
```

Comment:
I worry somewhat about this being too easy to destroy state; the reason one needs to specify `--force` to winget.exe.

Maybe it should be two parameter sets, a default set with `-Name` and another one with `-All`.  That way just running `> Reset-WinGetSource` doesn't immediately try to reset everything, but instead asks for a name.

Reply:
I created two parameters sets (DefaultSet and OptionalSet) so that we keep the default behavior of requiring the Name parameter. I also added an optional -All switch parameter that resets all sources but it is optional.

Reply:
Shouldn't `Name` and `All` be required within their respective parameter set?

Reply:
Removed brackets so they show up as being required.